src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -22,6 +22,23 @@
 from pants.rules.core.strip_source_root import SourceRootStrippedSources
 
 
+def get_timeout_seconds_for_target(test_target_timeout, timeout_default, timeout_maximum):
+  """Calculate the timeout for a test target.
+
+  If a target has no timeout configured its timeout will be set to the default timeout.
+
+  :param test_target: A test target
+  :param timeout_default: the default timeout in seconds.
+  :param timeout_maximum: the maximum timeout in seconds.
+  :return: timeout in seconds.
```

Comment:
I think it's fine to drop these as they don't add much new information from the already well-named params (especially when you add type hints). The one useful detail is that they use seconds.